controllers/admin/AdminFeaturesController.php

```diff
@@ -292,7 +292,7 @@ public function initToolbarTitle()
                 break;
 
             case 'view':
-                $bread_extended[] = $this->feature_name[$this->context->employee->id_lang];
+                $bread_extended[] = $this->feature_name[$this->context->employee->id_lang] ?? null;
```

Comment:
If the value is null, you will call $this->addMetaTitle(null);
Do you know what are the consequences ?

Reply:
yes, none, as there's a check `is_array` inside this method and `$bread_extended` is created a bit sooner with rest of the breadcrumbs displayed, I tested it before and after change and haven't noticed any differences 🤔